sql-cli/noxfile.py

```diff
@@ -25,37 +25,42 @@ def dev(session: nox.Session) -> None:
 def test(session: nox.Session, airflow: str) -> None:
     """Run both unit and integration tests."""
 
+    session.virtualenv.env.update(
+        {
+            # Airflow automatically creates the ~/airflow directory on importing airflow
+            # but we are not using this directory hence we change the home to a tmp directory to avoid side effects.
+            "AIRFLOW_HOME": session.create_tmp(),
```

Comment:
It's a shame we need to add this again :(

It would be great if we had some way of guaranteeing that when users are using the SQL CLI commands we're not depending on this home directory. Do you have any thoughts on this, @feluelle?

Reply:
I think it is possible to address this when using airflow as a library has been improved. The env var introduced in 2.4 is a great step in the right direction.

Reply:
Let me test using this in the `__init__.py` instead of in here.

Reply:
Init does not seem to be enough. It seems it needs to be set before installation 😅 or some other file/modules besides `__init__.py`.